repos/fdbt-site/src/pages/periodValidity.tsx

```diff
@@ -96,7 +99,24 @@ const PeriodValidity = ({ errors = [], fieldset, csrfToken }: PeriodValidityProp
                             <span className="govuk-hint" id="period-validity-hint">
                                 We need to know the time that this product would be valid until
                             </span>
+                            {errors.length > 0 && (
```

Comment:
Do we want this for all errors? What if they haven't selected the fare day option?